uw-frame-components/portal/timeout/services.js

```diff
@@ -11,8 +11,9 @@ define(['angular', 'jquery'], function(angular, $) {
     * getTimeout() - Gets the shib session, but returns just the timeout
                      and the time of that timeout.
     **/
-    .factory('PortalShibbolethService', ['$http', 'miscService', 'SERVICE_LOC',
-        function($http, miscService, SERVICE_LOC) {
+    .factory('portalShibbolethService', ['$http', '$log',
```

Comment:
> Shibboleth is awfully common among our target adopters.

Is uPortal-home targeting a specific subset of adopters?
The majority of uPortal adopters I am aware of do not use Shibboleth.

> In practice the Shib relationship is about headers, headers that are relatively easily placed by a variety of authentication environment solutions, including CAS.

Thinking even higher level, the primary function of this service appears to be providing userAttributes.
uPortal has a [user attributes abstraction](https://wiki.jasig.org/display/UPM43/User+Attributes).
Is this bypassing the abstraction, rather than leveraging it?